examples/jsm/renderers/webgpu/nodes/WebGPUNodeBuilder.js

```diff
@@ -100,6 +104,10 @@ class WebGPUNodeBuilder extends NodeBuilder {
 
 		this.uniformsGroup = {};
 
+		this.builtins = {
```

Comment:
Maybe it would be better to make it Set even now... I.e. `this.builtins = new Set()`, `this.builtins.add( 'instance_index' )`, and `if ( this.builtins.has( 'instance_index' ) )`.